cms/static/sass/elements/_system-feedback.scss

```diff
@@ -554,15 +562,35 @@
     }
 
     .action-primary {
-      @include blue-button();
+      @extend %btn-primary-blue;
+      @extend %sizing;
 
-      @extend %t-strong;
+      .action-button-text {
+        display: inline-block;
+        vertical-align: baseline;
+      }
 
-      border-color: $blue-d2;
+      .icon {
+        display: inline-block;
+        vertical-align: baseline;
+      }
+
+      // CASE: new/create button
+      &.new-button,
+      &.button-new {
+        @extend %btn-primary-green;
+        @extend %sizing;
+      }
     }
 
     .action-secondary {
       @extend %t-action4;
+      color: $white;
+      cursor: pointer;
+
+      &:hover {
+        color: $link-color;
```

Comment:
When I hover the mouse over "Dismiss", the color is too low contrast and unreadable. A lighter blue or something would be better.

![Screenshot 2023-09-13 at 11 33 56 AM](https://github.com/openedx/edx-platform/assets/945577/6ab8aa2c-d5ce-4adc-8784-058633e198e3)

Reply:
I see, I updated it to have a light gray color, since this also shows in the Advanced Settings warning mentioned below, it didn't look good with light blue. Here's how it looks now:

<img width="221" alt="Screen Shot 2023-09-14 at 12 47 17 PM" src="https://github.com/openedx/edx-platform/assets/6829768/9ade2678-6766-49cb-b183-d6a6fdf47489">

I did want to ask though, the screenshot you shared seems to have the old styling of the button, did it not show the new styles? You might need to run `paver compile_sass`, my bad I missed that step in the testing instructions.

Reply:
I had done that, but wasn't seeing them... am seeing it now. Must be some sort of cache issue. I'm not worried about it.